pynamodb/connection/base.py

```diff
@@ -427,11 +427,19 @@ def _make_api_call(self, operation_name, operation_kwargs):
                     'request_id': headers.get('x-amzn-RequestId')
                 }
 
-                if 'RequestItems' in operation_kwargs:
+                if REQUEST_ITEMS in operation_kwargs:
                     # Batch operations can hit multiple tables, report them comma separated
-                    verbose_properties['table_name'] = ','.join(operation_kwargs['RequestItems'])
+                    verbose_properties['table_name'] = ','.join(operation_kwargs[REQUEST_ITEMS])
+                elif TRANSACT_ITEMS in operation_kwargs:
+                    # Transactional operations can also hit multiple tables, or have multiple updates within
+                    # the same table
+                    items = []
+                    for item in operation_kwargs[TRANSACT_ITEMS]:
+                        for op in item.keys():
+                            items.append(json.dumps(item[op][TABLE_NAME]))
```

Comment:
You're just trying to quote them? Is `repr` easier?

Reply:
Do you mean item.__repr__() or item[op].__repr__()? Won't that be (potentially) too verbose, given that it will include the exact query/update conditions?

Reply:
I meant `item[op][TABLE_NAME]` which is, I assume, a string. I wondered whether the `json.dumps` around it is so that the list of tables `["foo", "bar", "baz"]` would turn into string `'"foo","bar","baz"`, and if so, would it be easier to do `repr(item[op][TABLE_NAME])`?

Reply:
Oh, nice catch. We don't need a json.dumps here. Was probably left over from some debugging I was doing.

Reply:
Looks like I messed something up really badly here. Hmm..

Reply:
Sorry, I'm new to github. :) 

I ended up creating a new pull request - https://github.com/pynamodb/PynamoDB/pull/835